exchanges/orderbook/orderbook_test.go

```diff
@@ -179,212 +168,171 @@ func TestGetOrderbook(t *testing.T) {
-	d, err := DeployDepth("test", c, asset.Spot)
+	d, err := DeployDepth("test", pair, asset.Spot)
 	require.NoError(t, err)
 	require.NotNil(t, d)
-	_, err = DeployDepth("test", c, asset.Spot)
+	_, err = DeployDepth("test", pair, asset.Spot)
 	require.NoError(t, err)
 }
 
 func TestCreateNewOrderbook(t *testing.T) {
-	c := currency.NewBTCUSD()
-	base := &Book{
-		Pair:     c,
+	pair := currency.NewBTCUSD()
+	b := &Book{
+		Pair:     pair,
 		Asks:     []Level{{Price: 100, Amount: 10}},
 		Bids:     []Level{{Price: 200, Amount: 10}},
 		Exchange: "testCreateNewOrderbook",
 		Asset:    asset.Spot,
 	}
 
-	err := base.Process()
-	require.NoError(t, err, "Process must not error")
-
-	result, err := Get("testCreateNewOrderbook", c, asset.Spot)
-	if err != nil {
-		t.Fatal("TestCreateNewOrderbook failed to create new orderbook", err)
-	}
+	require.NoError(t, b.Process(), "Process must not error")
 
-	if !result.Pair.Equal(c) {
-		t.Fatal("TestCreateNewOrderbook result pair is incorrect")
-	}
+	result, err := Get("testCreateNewOrderbook", pair, asset.Spot)
+	require.NoError(t, err)
+	assert.True(t, result.Pair.Equal(pair))
 
-	a, b := result.TotalAsksAmount()
-	if a != 10 && b != 1000 {
-		t.Fatal("TestCreateNewOrderbook CalculateTotalAsks value is incorrect")
-	}
+	ac, total := result.TotalAsksAmount()
+	assert.Equal(t, 10.0, ac, "TotalAsksAmount should return 10")
+	assert.Equal(t, 1000.0, total, "TotalAsksAmount should return 1000")
 
-	a, b = result.TotalBidsAmount()
-	if a != 10 && b != 2000 {
-		t.Fatal("TestCreateNewOrderbook CalculateTotalBids value is incorrect")
-	}
+	ac, total = result.TotalBidsAmount()
+	assert.Equal(t, 10.0, ac, "TotalBidsAmount should return 10")
+	assert.Equal(t, 2000.0, total, "TotalBidsAmount should return 2000")
 }
 
 func TestProcessOrderbook(t *testing.T) {
-	base := Book{
+	b := Book{
 		Asks:     []Level{{Price: 100, Amount: 10}},
 		Bids:     []Level{{Price: 200, Amount: 10}},
 		Exchange: "ProcessOrderbook",
 	}
 
 	// test for empty pair
-	base.Pair = currency.EMPTYPAIR
-	err := base.Process()
+	err := b.Process()
 	assert.ErrorIs(t, err, errPairNotSet)
 
 	// test for empty asset type
-	c := currency.NewBTCUSD()
-	base.Pair = c
-	err = base.Process()
-	if err == nil {
-		t.Error("empty asset type should throw an err")
-	}
+	pair := currency.NewBTCUSD()
+	b.Pair = pair
+	err = b.Process()
+	require.ErrorIs(t, err, errAssetTypeNotSet)
 
 	// now process a valid orderbook
-	base.Asset = asset.Spot
-	err = base.Process()
-	if err != nil {
-		t.Error("unexpected result: ", err)
-	}
-	result, err := Get("ProcessOrderbook", c, asset.Spot)
-	if err != nil {
-		t.Fatal("TestProcessOrderbook failed to create new orderbook")
-	}
-	if !result.Pair.Equal(c) {
-		t.Fatal("TestProcessOrderbook result pair is incorrect")
-	}
+	b.Asset = asset.Spot
+	require.NoError(t, b.Process(), "Process must not error")
+
+	result, err := Get("ProcessOrderbook", currency.NewBTCUSD(), asset.Spot)
+	require.NoError(t, err)
```

Comment:
🐒 Conversely Unit test subject (`Process`) doesn't need a comment, but assistance call `Get` should
```suggestion
	require.NoError(t, b.Process(), ", "Process must not error")

	result, err := Get("ProcessOrderbook", currency.NewBTCUSD(), asset.Spot)
	require.NoError(t, err, "Get must not error")
```